### DISCUSSION TITLE
w3m button styling the text color

### DISCUSSION THREAD

**Discussion by jennyunh** [reown-com/appkit#2513] [Q&A] [answered]
We are using the w3m-button: https://docs.walletconnect.com/appkit/react/core/components

I've noticed that there is no option to adjust the button's text color.
In some other doc I saw that there is a --w3m-accent-fill-color option that allows you to change the button text color, 

but currently this option does not exist:
https://docs.walletconnect.com/appkit/react/core/theming

Is there no way to change the button text color?

**Comment by DevMarc16** [comment]
same issue lol

**Comment by rtomas** [comment, marked as answer]
Please check how to style the buttons -> https://docs.walletconnect.com/appkit/react/core/theming
or use a custom one -> https://github.com/WalletConnect/web-examples/blob/main/dapps/web3modal/next/components/CustomButton.tsx
